<launch>
    <node pkg="sgr_project" type="smartband_receiver.py" name="smartband_receiver" output="screen" />
</launch>